packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -152,6 +152,24 @@ export class Blockchain {
   async getRawMempool (verbose: boolean): Promise<string[] | MempoolTx> {
     return await this.client.call('getrawmempool', [verbose], 'bignumber')
   }
+
+  /**
+   * Get block statistics for a given window.
+   * @param {number} hashOrHeight  The block hash or height of the target block.
```

Comment:
If you read other's code there is always a break line between comment and param sections